src/Symfony/Bundle/WebProfilerBundle/EventListener/WebDebugToolbarListener.php

```diff
@@ -112,21 +112,30 @@ public function onKernelResponse(FilterResponseEvent $event)
     protected function injectToolbar(Response $response, Request $request)
     {
         $content = $response->getContent();
-        $pos = strripos($content, '</body>');
-
-        if (false !== $pos) {
-            $toolbar = "\n".str_replace("\n", '', $this->twig->render(
-                '@WebProfiler/Profiler/toolbar_js.html.twig',
-                array(
-                    'position' => $this->position,
-                    'excluded_ajax_paths' => $this->excludedAjaxPaths,
-                    'token' => $response->headers->get('X-Debug-Token'),
-                    'request' => $request,
-                )
-            ))."\n";
-            $content = substr($content, 0, $pos).$toolbar.substr($content, $pos);
-            $response->setContent($content);
+        $canInjectToolbar = false !== stripos($content, '<title>');
+        $injectToolbarAt = strripos($content, '</body>');
+
+        $canInjectToolbar |= false !== $injectToolbarAt;
```

Comment:
using a binary operator here looks wrong to me, as ``$canInjectToolbar`` is an integer after than, not a boolean.